macros/src/lib.rs

```diff
@@ -492,7 +499,10 @@ pub fn pre_init(args: TokenStream, input: TokenStream) -> TokenStream {
 
 // Creates a random identifier
 fn random_ident() -> Ident {
-    let mut rng = rand::thread_rng();
+    let seed = SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_secs();
+    CALL_COUNT.fetch_add(1, Ordering::SeqCst);
+    let seed = [seed, CALL_COUNT.load(Ordering::SeqCst) as u64];
+    let mut rng = rand::rngs::SmallRng::from_seed( unsafe { mem::transmute(seed) } );
```

Comment:
Let's not use `unsafe` code here. There are methods to go from `u64` to `[u8; 8]` and vice versa that could be used here.

Reply:
To the best of my knowledge only in nightly...
https://doc.rust-lang.org/nightly/std/primitive.u64.html#method.to_be_bytes

Of course I could do some bitmasking. ;)

Reply:
> To the best of my knowledge only in nightly...

Still? I thought it was already stabilized. Perhaps I was thinking of `f32.to_bits`.

> Of course I could do some bitmasking. ;)

I would prefer that over unsafe code.